inc/item_ticket.class.php

```diff
@@ -411,7 +411,7 @@ static function showForTicket(Ticket $ticket) {
             self::dropdownMyDevices($dev_user_id, $ticket->fields["entities_id"], null, 0, ['tickets_id' => $instID]);
          }
 
-         $data =  array_keys(getAllDatasFromTable('glpi_items_tickets'));
+         $data =  array_keys(getAllDataFromTable('glpi_items_tickets'));
```

Comment:
This is not related to this PR, but, I think usage of `array_keys` is wrong here, as on line 418, logic tries to get `itemtype` ad `id` for each element of `$data`.

Reply:
Indeed, I'll fix

Reply:
In fact, fix is more complex. See https://github.com/glpi-project/glpi/pull/5858 .
If we use all data from the table, it will not be possible to associate an item to more than one ticket (even closed).

Reply:
I've reverted my commit